compiler/front/msgs.nim

```diff
@@ -590,31 +590,31 @@ proc joinAnyOf*[T](values: seq[T], quote: bool = false): string =
     else:
       s
 
-  if len(values) == 1:
+  case len(values)
+  of 1:
     result.add q($values[0])
-  elif len(values) == 2:
+  of 2:
     result.add q($values[0]) & " or " & q($values[1])
   else:
-    for idx in 0 ..< len(values) - 1:
```

Comment:
I think `..<` is preferable even with a suffixed `-1` since it's clearer that it's `1` last element we skip.

Reply:
Yes, I also think `..<` is more approachable. Or use `high(values) - 1`, because we have a specific "last existing index" function for almost all containers anyway.

Reply:
done.